cockroachdb: support all v23.2 metrics (#16697)
DataDog/integrations-core: cockroachdb/cmd/generate/README.md

# generate

A program to generate the `metadata.csv` and metric name mapping required by the
`cockroachdb` Datadog integration.

## Usage

Build the `generate` program.

```
go build .
```

Download the `metrics.html` from
https://github.com/cockroachdb/cockroach/blob/master/docs/generated/metrics/metrics.html.

Run the generator against the `metrics.html` file.

```
generate metrics.html
```

> Afterwards, you'll have a `metadata.csv` and `mapping.txt` file in your current

Reviewer: ```suggestion
A `metadata.csv` and `mapping.txt` file is generated in your current
```
